commandline.c

```diff
@@ -4,12 +4,14 @@
 
 #include "commandline.h"
 
+#include <getopt.h>
```

Comment:
You've decided to use getopt and not getopt_long. So please include unistd.h instead.